kernel/declareops.ml

```diff
@@ -289,9 +289,9 @@ let hcons_mind mib =
 (** {6 Stm machinery } *)
 
 let string_of_side_effect { Entries.eff } = match eff with
```

Comment:
Should this function also get `debug` prefix?

Reply:
If the `debug` prefix is definitively adopted then ok to generalize it. I did not do it because I was unsure `debug` was really the best terminology.

An other alternative to the `debug` prefix could be `basic`, or something on this base. What do you think?